Gemfile

```diff
@@ -99,7 +99,7 @@ gem 'less-rails-bootstrap', '~> 3.3.5'
 gem 'compass-rails', '~> 2.0.5'
 gem 'compass-blueprint', '~> 1.0.0'
 gem 'jquery-ui-rails', '~> 6.0.1'
-gem 'jquery-rails', '~> 3.1.4'
+gem 'jquery-rails', git: 'https://github.com/viclim/jquery-rails', ref: '3d9985112690c68552a8a10abfdb5ea80b40dec9'
```

Comment:
vimlim/jquery-rails branches from indirect/jquery-rails, not rails/jquery-rails...any reason why?

Reply:
@nickmarden I forked it really long time ago from there. Let me see if I can correct it.

Reply:
Okay. It's now forked from rails/jquery-rails.

Reply:
@viclim could you please use https://github.com/GSA/jquery-rails/commit/c0c56208f57746a4bc63431d5e771cea4486efca instead? It includes your changes but lives in a GSA-owned fork which is probably better in terms of maintainability.